README.md

```diff
@@ -105,35 +83,7 @@ The results from the scan are delivered by webhook; for more information about s
 
 #### Sample Code
 
-```java
-// By default, the client reads the API key from the environment variable NIGHTFALL_API_KEY
-try (NightfallClient c = NightfallClient.Builder.defaultClient()) {
-
-    // Define some detectors to use to scan your data
-    Detector creditCard = new Detector("CREDIT_CARD_NUMBER");
-    creditCard.setMinConfidence(Confidence.LIKELY);
-    creditCard.setMinNumFindings(1);
-    Detector ssn = new Detector("US_SOCIAL_SECURITY_NUMBER");
-    ssn.setMinConfidence(Confidence.POSSIBLE);
-    ssn.setMinNumFindings(1);
-
-    // A rule contains a set of detectors to scan with
-    DetectionRule rule = new DetectionRule(Arrays.asList(creditCard, ssn), LogicalOp.ANY);
-
-    // File scans are conducted asynchronously, so provide a webhook route to an HTTPS server to send results to.
-    String webhookResponseListenerURL = "https://my-service.com/nightfall/listener";
-    ScanPolicy policy = ScanPolicy.fromDetectionRules(Arrays.asList(rule), webhookResponseListenerURL);
-    ScanFileRequest req = new ScanFileRequest(policy, "my request metadata");
-
-    // Upload the data to the API, then trigger the async scan
-    File file = new File("./super-secret-credit-cards.pdf");
-    try (InputStream stream = new FileInputStream(file)) {
-        ScanFileResponse response = c.scanFile(req, stream, file.length());
-        System.out.println("started scan: " + response.toString());
-    }
-}
-```
-
+see [the FileScannerExample class](src/main/java/ai/nightfall/examples/FileScannerExample.java)
```

Comment:
Do we have any docs on how to create a webhook server and get the URL? Maybe @Dhertz knows. 

I think it would be helpful for devs that haven't used webhooks (like me) if we also have a "usage" code block here or in the javadocs that sets the `NIGHTFALL_API_KEY`, starts a webhook server, and passes in the url and a filename as args, and has the exact command to start the main class.

Reply:
yeah.. I like Dan's minimal Go example, maybe we just link to a Python or Go server? I think webhooks are a little trickier without external dependencies, that would be an argument for an additional source set (so our published artifact doesn't have the examples or server or their dependencies). Here's a Spring Boot example: https://www.twilio.com/blog/coding-twilio-webhooks-in-java-with-spring-boot